guides/data_and_structure.md

```diff
@@ -0,0 +1,151 @@
+# Data Model and Structure
+
+## Content
+
+#### Hierarchy
+
+- Network
+	- Podcast
+		- Episode
+			- Audio
+	- Audio
+
+
+
+### Network
+
+Field   | Type     | Description 
+------: | :--------| :-----------
+`title` | `String` | The name of the network
+`image` | `Image`  | An avatar style image representing the network
+ 
+ 
+Potential fields open for discussion: 
+
+* `slug` 
+* `link`
+
+### Podcast
+
+A **Podcast** belongs to exactly a **Network**
+
+ Field | Type | Description 
+   --: | :--  | :--
+`shortID` | `String` | Short basis to identify a podcast in an URL, hashtag, etc. Also for easy referring to episodes. E.g. `FS` for freakshow, so an episode becomes `FS130`. Typically between 2 and 5 letters, and has to be unique inside a network as it is used for URL slugs as well.
+ `title` | `String` | The name of the podcast
+ `image` | `Image`  | An cover image for the podcast
+ `subtitle` | `String` | The subtitle line. Is put in `description` and `itunes:subtitle` in the feed. Usually used as one line description display in podcast directories. Currently doesn't appear in iTunes anywhere prominently.
+ `summary` | `String` | A longer description of the podcast. Is put in `itunes:summary` in the feed. Is displayed in the iTunes Podcast preview.
+`author` | `String` | The Author of this podcast. Prominently displayed by all directories and live players. Put in `itunes:author` for the channel.
+`ownerName` | `String` | Owner of the podcast. Put into `itunes:owner` `itunes:name` in the feed
+`ownerEmail` | `String` | Owner of the podcast. Put into `itunes:owner` `itunes:email` in the feed - both name and email mostly used for administrative uses - directories e.g. don't expose them.
+
+Important fields currently missing: 
+
+* `language` - [ISO 639-1](http://www.loc.gov/standards/iso639-2/php/code_list.php)
+
+Potential fields open for discussion:  
+
+* `link`
+* More iTunes metadata fields  (`itunes:explicit`, `itunes:category`)
+* Contributors
+
+Existing fields, worthy of discussion:
+
+* `publishedAt` - is currently both put into the feeds `pubDate` tag (which is quite unused by clients/directories) and used as a switch to say if a podcast is public. @monkeydom: the removing the `publishedAt` would be prudent, as it is ambiguous, and the mechanism to determine the published state of a podcast should be handled differently.
```

Comment:
No harm in filling the `pubDate` field thoug? But I agree, the publish state should be handled separately.

Reply:
Went through the specs, didn't find any reference to it. I think it is ambiguous, and at best should just default to the first episode ever published probably. Nobody is using the data, so I think removing it is nicer, just unused unclear semantics otherwise.

Reply:
Using publishing date of first episode sounds good. My +1 for removing it then :)